wiki/Tournaments/oPHN/2023_catch/en.md

```diff
@@ -0,0 +1,95 @@
+- [Twitch](https://www.twitch.tv/osuliveph)
+- [Youtube](https://www.youtube.com/channel/UCkV4V1OVO6gOGxtH51glH0Q/live)
+
+## Rulesets
+
+### General rules
+
+- Score V2 will be used
+- NoFail will be enforced regardless of the mod being used per map.
+
+### Qualifier stage
+
+- There will be 5 maps, consisting of 2 NoMod, 1 Hidden, 1 HardRock and 1 DoubleTime.
+- Players will be scheduled to a multiplayer room according to their availability.
+- Each player will get 2 attempts at each map, only the best attempt for each map will count
+- Each map will have its own leaderboard, and each player's overall seed will be determined by the average of these ranks.
+- The top 16 players advance to the Bracket Stage.
+
+### Bracket stage
+
+- Players are seeded into a double-elimination bracket, which would consist of the top 16 players of the qualifier round.
+- Mappool Sizes and Best of Format for each round are given as follows:
+
+| Round stage | Best of | NM | HD | HR | DT | TB |
+| :-- | :-: | :-: | :-: | :-: | :-: | :-: |
+| Round of 16 | 9 | 4 | 2 | 2 | 2 | 1 |
+| Quarterfinals | 9 | 4 | 2 | 2 | 2 | 1 |
+| Semifinals | 11 | 4 | 3 | 3 | 3 | 1 |
+| Finals | 11 | 4 | 3 | 3 | 3 | 1 |
+| Grand finals | 13 | 5 | 3 | 3 | 3 | 1 |
+
+### Match rules
+
+- The multiplayer room will be created and invites sent to players 5-10 minutes prior to the match start time.
+- If a player is more than 10 minutes late to the for their match, the player inside the MP room will be declared the winner. If neither player appears, the player with the higher seed will be considered the winner.
+- Each player will be allowed to select 1 map as a warmup. Warmups must be uploaded to the osu! servers and cannot exceed 4 minutes in length.
+- Players can choose to ban 1 map from the mappool (excluding the tiebreaker). The pick/ban order is determined by the !roll command. The player with the lower roll bans first and the player with the higher roll picks first. Players will alternate picks until a match winner is decided.
+- If both players are tied on match point (e.g. 4-4 score in a Best-of-9), the players will play the tiebreaker map of the pool to determine the winner.
+- Players are given 2 minutes to ban/pick maps when it is their turn. Failure to do so means that the referee will ban/pick at random in behalf of the player.
+- Players are eligible to use a 2 minute tactical time out once per match. They are only allowed to use it when it is their turn to pick. It will take into effect after the pick timer has expired.
+
+### Player eligibility
+
+**Players who are ranked #5000 and above in the osu!catch global rankings will be allowed to join the tournament. Additionally, players who currently have (or had at some point since Account Creation) the Philippine Flag on their profile or players who do not have the Philippine Flag on their profiles but can present a proof of their Philippine Citizenship to the staff are the only ones allowed to join the tournament.**
```

Comment:
```suggestion
Players who are ranked #5000 and above in the osu!catch global rankings will be allowed to join the tournament. Additionally, players who currently have (or had at some point since their account's creation) the Philippine flag on their profile, or players who do not have the Philippine flag on their profiles but can present a proof of their Philippine citizenship to the tournament staff are the only ones allowed to join the tournament.
```